src/constants.ts

```diff
@@ -77,14 +78,18 @@ export const ReasonNftAvatar: number = 1;
 export const ReasonBusinessVerified: number = 2;
 export const ReasonTransphobia: number = 3;
 export const ReasonPromoted: number = 4;
+export const ReasonDisallowedWordsOrEmojis: number = 5;
 export const ReasonMap = {
 	[ReasonBlueVerified]: 'Twitter Blue verified',
 	[ReasonNftAvatar]: 'NFT avatar',
 	[ReasonBusinessVerified]: 'Twitter Business verified',
 	[ReasonTransphobia]: 'transphobia',
 	[ReasonPromoted]: 'promoting tweets',
+	[ReasonDisallowedWordsOrEmojis]: 'disallowed words or emojis',
 };
 
+export const emojiRegExp = RegExp(/(\u00a9|\u00ae|[\u2000-\u3300]|\ud83c[\ud000-\udfff]|\ud83d[\ud000-\udfff]|\ud83e[\ud000-\udfff])/, 'g')
```

Comment:
is this reliable? where did you find this?

Reply:
https://stackoverflow.com/questions/60626009/regex-filter-special-characters-like-japanese-but-preserve-emojis

Reply:
that question specifies that that regex string captures more than just emoji characters, specifically japanese characters. the answer links to this repo: https://github.com/mathiasbynens/emoji-regex which we could use, or the one that I provided previously

Reply:
The one you wrote up failed for emojis that are an emoji + zero width combiner + modifier :(

I also saw the emoji-regex package, my one concern is that the pattern by itself is 12.8 kB which uhhhh, suckssssss :(

Reply:
`[\p{Emoji_Presentation}\u200d]+/gu`? which emojis were you testing with? I was using the family ones

https://regex101.com/r/E4rmCw/1

Reply:
oh huh, weird now it works, ok I'll switch it over

Reply:
I was using 🏋️‍♂️, it still seems like it's only matching the zero-width and not the underlying weight lift emoji? But that's still better than matching Japanese